htdocs/societe/paymentmodes.php

```diff
@@ -1579,6 +1582,7 @@
 		print_liste_field_titre($form->textwithpicto($langs->trans('ExternalSystemID'), $langs->trans("IDOfPaymentInAnExternalSystem")));		// external system ID
 		//print_liste_field_titre("Bank");
 		print_liste_field_titre("RIB");
+		print_liste_field_titre("CCI");
```

Comment:
I think the code CCI is used by Peru only, right ?
If yest, can you add a condition
`if ($mysoc->country_code == 'PE) {`
'